### ISSUE TITLE
How to use public OpenAI APIs ?

### ISSUE THREAD

**Issue by Roydon** [microsoft/sample-app-aoai-chatGPT#307]
I don't have access to Azure OpenAI service and would like to use public OpenAI APIs for this Sample App. Can anyone suggest if this is possible ?

**Comment by gopi-hl** [comment]
No. This is not possible on this one. However, do checkout the Semantic Kernel library and samples. That one can. https://github.com/microsoft/chat-copilot

**Comment by pamelafox** [comment]
There's also support for non-Azure OpenAI in this sample, see instructions here: https://github.com/Azure-Samples/azure-search-openai-demo/#existing-openai-resource
